packages/fuels-contract/src/script.rs

```diff
@@ -812,3 +815,110 @@ mod test {
         Bech32ContractId::new("fuel", rand::thread_rng().gen::<[u8; 32]>())
     }
 }
+
+#[derive(Default)]
+pub struct ScriptBuilder {
+    gas_price: Word,
+    gas_limit: Word,
+    maturity: Word,
+    receipts_root: Bytes32,
+    script: Vec<u8>,
+    script_data: Vec<u8>,
+    inputs: Vec<Input>,
+    outputs: Vec<Output>,
+    witnesses: Vec<Witness>,
+    metadata: Option<Metadata>,
+    asset_id: AssetId,
+    amount: u64,
+}
+
+impl ScriptBuilder {
+    pub fn new() -> ScriptBuilder {
+        ScriptBuilder {
+            gas_price: 0,
+            gas_limit: 0,
+            maturity: 0,
+            receipts_root: Default::default(),
+            script: vec![],
+            script_data: vec![],
+            inputs: vec![],
+            outputs: vec![],
+            witnesses: vec![],
+            metadata: None,
+            asset_id: AssetId::default(),
+            amount: 0,
+        }
```

Comment:
I _think_ this can be replaced with an `impl Default`?

Reply:
We can do it, but I don't think we need it.